lib/models/command.js

```diff
@@ -127,14 +127,14 @@ Command.prototype.validateAndRun = function(args) {
 
   if (this.works === 'insideProject' && !this.isWithinProject) {
     return Promise.reject(new SilentError(
-      'You have to be inside an ember-cli project in order to use ' +
-      'the ' + chalk.green(this.name) + ' command.'
+      chalk.yellow('You have to be inside an ember-cli project in order to use the ') +
+      chalk.green(this.name) + chalk.yellow(' command.')
```

Comment:
well, actually i just saw those other occurrences were also introduced by my PR (i honestly can't remember why i did this, maybe it made sense before i rebased for some reason). i will revert those changes.